lib/matplotlib/rcsetup.py

```diff
@@ -1313,6 +1313,7 @@ def validate_animation_writer_path(p):
     'keymap.quit':         [['ctrl+w', 'cmd+w', 'q'], validate_stringlist],
     'keymap.quit_all':     [['W', 'cmd+W', 'Q'], validate_stringlist],
     'keymap.grid':         [['g'], validate_stringlist],
+    'keymap.grid_both':    [['G'], validate_stringlist],
```

Comment:
Is it normal that it is `'keymap.grid_both'` here while it is `'keymap.grid_minor'` anywhere else in the commit?